lua/nvim-highlight-colors/colors.lua

```diff
@@ -3,7 +3,7 @@ local table_utils  = require("nvim-highlight-colors.table_utils")
 
 local M = {}
 
-M.rgb_regex = "rgba?[(]+" .. string.rep("%s*%d+%s*", 3, ",") ..",?%s*%d*%.?%d*%s*[)]+"
+M.rgb_regex = "rgba?[(]+" .. string.rep("%s*%d+%s*", 3, "[, ]") .. "[, /]?%s*%d*%.?%d*%s*[)]+"
```

Comment:
I think white-space would be better visualized if we use %s instead.
```suggestion
M.rgb_regex = "rgba?[(]+" .. string.rep("%s*%d+%s*", 3, "[,%s]") .. "[, /]?%s*%d*%.?%d*%s*[)]+"
```